.rubocop_todo.yml

```diff
@@ -15,6 +15,12 @@ Metrics/AbcSize:
 # ExcludedMethods: refine
 Metrics/BlockLength:
   Max: 89
+  Exclude:
```

Comment:
So instead of keep adding to the rubocop_todo config, why not try to settle on better defaults. Something like this in _rubocop.yml_ would work for me:

```yaml
Metrics/ModuleLength:
  Max: 200

Metrics/BlockLength:
  Max: 50
  ExcludedMethods: ['describe', 'context']
```

I find the default values to low and I don't think it is worth trying to refactor just to meet these defaults. A max module length of 200 and a max block length of 50 seems reasonable to me. 
Also imo it makes sense to exclude describe and context which would mainly match the minitests. 

If we go down this route, we can remove BlockLength and ModuleLength from the rubocop_todo. WDYT?

Reply:
I think that lowering BlockLength (from 89 to 50) *and* excluding describe and context are a nicer solution.
For the ModuleLength I think that excluding tests makes more sense than increasing the limit.